ormolu-live/src/Main.hs

```diff
@@ -96,6 +96,8 @@ viewModel model@Model {..} =
   div_
     []
     [ link_ [rel_ "stylesheet", href_ "bulma.min.css"],
+      script_ [src_ "clipboard.min.js"] "",
+      script_ [] "var c = new ClipboardJS('.copy-output')",
```

Comment:
I think this is fine, invoking this via JSaddle would be more ceremony without much benefit. Small suggestion:
```suggestion
      script_ [] "new ClipboardJS('.copy-output');",
```

Reply:
Huh, I could have sworn I tried not assigning the `ClipboardJS` object to a variable & it didn't work, but I've just tried it again now and it works fine. Will update now, thanks!